crates/cli/tests/cli_run.rs

```diff
@@ -511,6 +551,74 @@ mod cli_run {
         )
     }
 
+    #[test]
+    #[cfg_attr(windows, ignore)]
+    fn expects_dev_and_test() {
+        // these are in the same test function so we don't have to worry about race conditions
+        // on the building of the platform
+
+        test_roc_app(
+            "crates/cli_testing_examples/expects",
+            "expects.roc",
+            "expects",
+            &[],
+            &[],
+            &[],
+            indoc!(
+                r#"
+                This expectation failed:
+
+                14│      expect x != x
+                                ^^^^^^
+
+                When it failed, these variables had these values:
+
+                x : Num *
+                x = 42
+
+                [<ignored for tests> 15:9] 42
+                [<ignored for tests> 16:9] "Fjoer en ferdjer frieten oan dyn geve lea"
```

Comment:
Google Translate says: "Fire and destroy fries to your given body" 🔥 🍟

Reply:
it's from this song https://www.youtube.com/watch?v=-uafukuUKy0 which is an ode to the island of https://en.wikipedia.org/wiki/Heligoland. 

it is prose, so I understand how google translate is not ... great at it. something like  "the elements wore away your body" captures the meaning  better, though lots of subtlety is lost. More litterally, maybe "fire and perdition ate away at your full body".